pipeline/extract_carbon.py

```diff
@@ -11,6 +11,10 @@
 import config as cg
 from constants import Constants as ct
 
+save_directory = 'tmp/data'
```

Comment:
This is good for now but we should probably put this in config.py and have it try to build the needed directories at the start of our extract scripts.